src/webgl/p5.RendererGL.js

```diff
@@ -2463,6 +2465,23 @@ p5.RendererGL = class RendererGL extends Renderer {
     return tessy;
   }
 
+  _updateTessyCombineCallback() {
+    // If custom attributes have been used, the vertex data which needs to be 
+    // combined has changed, so libtess must have its combine callback updated
+    const combinecallback = (coords, data, weight) => {
```

Comment:
I'm just comparing this function to the initial combine callback we pass in, is the code the same or is there a part of it that's updated?
https://github.com/processing/p5.js/blob/8f463374cc33932f75931ac1a89e32ff63273559/src/webgl/p5.RendererGL.js#L2451-L2460

Reply:
It is the same, it's just that the result array size is different:
```js
const result = new Array(p5.RendererGL.prototype.tessyVertexSize).fill(0); // original
const result = new Array(this.tessyVertexSize).fill(0);  // when it needs to be updated
```

Reply:
This has been changed now!